generators/server/templates/src/main/docker/keycloak.yml.ejs

```diff
@@ -21,16 +21,13 @@ version: '<%= DOCKER_COMPOSE_FORMAT_VERSION %>'
 services:
   keycloak:
     image: <%= DOCKER_KEYCLOAK %>
-    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Djboss.socket.binding.port-offset=1000", "-Dkeycloak.profile.feature.upload_scripts=enabled"]
+    command: ["start-dev", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Dkeycloak.profile.feature.scripts=enabled"]
     volumes:
       - ./realm-config:/opt/jboss/keycloak/realm-config
     environment:
-      - KEYCLOAK_USER=admin
-      - KEYCLOAK_PASSWORD=admin
-      - DB_VENDOR=h2
```

Comment:
what's the default database in new distribution?

Reply:
h2

Reply:
In that case, I will prefer to add the statement back so that it's explicit to end users and they should manually switch to some other db for production use

Reply:
@vishal423 keycloak has changed a lot, and to use it without start-dev you will need much more than juste change the db type:

https://www.keycloak.org/migration/migrating-to-quarkus#_quarkus_is_not_an_application_server

any one using keycloak in production should be familiar with this.

Reply:
I can understand, but it would be good to make such things explicit. 

Since this upgrade is a major change on the keycloak side and requires a different configuration than before, it would be good to include a few documentation references to guide end-users for best production deployment recommendations.

Reply:
I ll try to update the docs before the issue with import is fixed